src/appleseed/renderer/modeling/edf/osledf.cpp

```diff
@@ -213,6 +214,24 @@ namespace
             return probability;
         }
 
+		float get_uncached_max_radiance_value() const
```

Comment:
Add `APPLESEED_OVERRIDE` at the end of the method to tell the compiler and the reader that this method is supposed to override another.